nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,269 @@
+
+![](./images/git-data-source_13.png)
+
+Once the repository syncs the details can be found in the **Synchronization Status** tab.  For example the platform specifics were synced.
+
+![](./images/git-data-source_14.png)
+
+The repository structure is:
+```
+▶ tree config_contexts 
+config_contexts
+├── devices
+│   ├── site-a-bb-01.yml
+│   ├── site-a-rtr-01.yml
+│   ├── site-a-rtr-02.yml
+│   ├── site-a-spine-01.yml
+│   ├── site-a-spine-02.yml
+│   ├── site-b-bb-01.yml
+│   ├── site-b-leaf-01.yml
+│   ├── site-b-leaf-02.yml
+│   ├── site-b-rtr-01.yml
+│   ├── site-b-rtr-02.yml
+│   ├── site-b-spine-01.yml
+│   └── site-b-spine-02.yml
+├── platform_eos.yml
+├── platform_junos.yml
+├── platform_nxos.yml
+└── role_spine.yml
+
+1 directory, 16 files
+```
+
+Configuration Context details:
+- Follows an inheritance methodology similar to what Ansible implements.  Global contexts can be overwritten by local contexts at both a group level, as well as at a device specific level.
+- Nautobot UI provides a simple view to see merged config contexts.  It can be visualized by navigating to a device and clicking on the **config contexts** tab.
+
+Here's an example, with some of the details omitted for brevity.
+
+![](./images/git-data-source_15.png)
+
+There is a huge benefit to having `config contexts` managed by a Git workflow.  This type of data can be modified often, especially platform specifics, or new device roles.  Utilizing a standard Git workflow allows for all the proper reviews and approvals to be accomplished before accepting the changes into Nautobot for use.
+
+
+## Additional Git Data Sources
+
+As seen in [Step 2](#Step-2), the standard installation of Nautobot will come natively with export templates, jobs, and config contexts.  Additional data sources can be incorporated using the Nautobot plugin system.  For example, the [nautobot-plugin-golden-config](https://github.com/nautobot/nautobot-plugin-golden-config) plugin implements four additional data sources.
+
+- Config Contexts
+- Backup Configs
+- Intended Configs
+- Jinja Templates
+
+For more information for the Golden Configuration specific data sources, navigate to [Nautobot Golden Config Repo](https://github.com/nautobot/nautobot-plugin-golden-config/blob/develop/docs/navigating-golden.md#git-settings).
+
+
+## Common Issues and Troubleshooting
+1. Repository is linked, but data is not properly loaded into Nautobot.
+    - Validate the root directory is set to the proper name.
+    - Export Templates -> `export_templates`.
+    - Jobs -> `jobs`.
+    - Config Contexts -> `config_contexts`.
+2. Synchronization Status Failures.
+    - Validate branch is correct and exists in the remote repository.
+    ![](./images/git-data-source_16.png)
+    - Validate remote url is correct and is the `https` url.  `ssh` urls are not currently supported.
+    ![](./images/git-data-source_17.png)    
```

Comment:
This image is showing a "Could not read Username" error - is that the correct intended image for this item?

Reply:
it was the wrong image, great catch.